includes/class-llms-blocks-migrate.php

```diff
@@ -163,8 +163,6 @@ private function get_template( $post_type, $merge_deprecated_versions = false )
 			<?php endif; ?>
 
 <!-- wp:llms/course-continue-button -->
-<div class="wp-block-llms-course-continue-button" style="text-align:center">[lifterlms_course_continue_button]</div>
```

Comment:
can you use the same logic as above?
when `$merge_deprecated_versions` is true?
See why we have that param looking at the param description :D